org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/i18n/I18nBase.java

```diff
@@ -14,6 +14,7 @@
 
   private Locale locale;
   private ResourceBundle i18nMessages;
+  private boolean warnAboutMissingMessages = false;
```

Comment:
This should not be false be default. The whole issue here was that I wasn't notified of errors in localization when I was debugging an issue earlier. If a user of the library wants to suppress being told that there are issues with localization, that's fine by me. But the default should always be to inform. 

Maybe modify the error message to say something like "...if this is expected, and you would like to disable notifications for non-existant strings in your selected local..."

Reply:
hmm. actually it works the way you want it's just named the wrong way around.

Reply:
oops no. I just get the default around the wrong way in my head